Add “An Unknown Friend”

robinwhittleton opened the pull request:
A [new Bunin short](https://hdl.handle.net/2027/uva.x030236881?urlappend=%3Bseq=295) became available this year, from a 1925 issue of _The Adelphi_. This PR adds that story, along with all of the associated metadata changes.

The story has a new translator, which means that the URL will change. I’ve prepped a PR for web for that.

robinwhittleton committed:
Add “An Unknown Friend”

robinwhittleton committed:
[Editorial] Modernize hyphenation and spelling in “An Unknown Friend”

robinwhittleton committed:
Add new metadata for “An Unknown Friend”

acabal commented:
Thanks!

Since you're adding a line to the titlepage, I think the canvas needs to be extended too. The easiest way to do this is to just run create-draft into a dummy folder and copy the generated titlepage to the real folder.

Also, the id.loc.gov links should not have https, because the official URIs are listed as http. Annoying but that's what the URI is.

robinwhittleton committed:
Regenerate the titlepage

…using a fresh create-draft.

robinwhittleton commented:
Added the new titlepage, and rebased away the https references. I’ve been using those recently as Wikipedia has them in their authority control section. I’ll fix the couple of projects I’ve got in flight at the moment, but should we add that as a lint rule?

acabal commented:
Yeah that would be an easy check to do, if you want to take care of it.

Thanks for the fixes!
